siriuspy/siriuspy/pwrsupply/bsmp/commands.py

```diff
@@ -574,7 +574,8 @@ def sofb_ps_func_return(self):
     def sofb_ps_setpoint_set(self, value):
         """."""
         if self._sofb_ps_setpoint is None:
-            self._sofb_ps_setpoint = _np.zeros(value.size, dtype=float)
```

Comment:
@fernandohds564 , this const has always the same value, 4. This is just the maximum number of BSMP devices that each UDC that we have today can control. This value is 4 even if UDC controls 3 or less devices.

Reply:
I know @xresende, that's why we always send setpoints with arrays of size four.